src/utils/tasks.js

```diff
@@ -71,7 +71,8 @@ function jobMessageOutputs (jobNode) {
   const ret = []
   let messageOutput
 
-  for (const message of jobNode.node.messages || []) {
+  for (const messageString of jobNode.node.messages || []) {
+    const [level, message] = messageString.split(':')
```

Comment:
Unfortunately as Oliver pointed out to me, we can't use `String.split()` as there could be multiple colons after the first severity level one. We'll have to use a regex here, I think something like
```js
/^(?:(DEBUG|INFO|WARNING|ERROR|CRITICAL):)?(.*)/
```